config/locales/en.yml

```diff
@@ -1691,6 +1693,67 @@ en:
             allow_registration_without:
               "true": "Competitors will be allowed to register for events they're not qualified for (I'll remove them later)"
               "false": "Competitors MUST be qualified to register for an event"
+        #context: new registration system
+    registration_v2:
+      pdf:
+        #context: <0> indicates an html tag for internationalizing in React, see https://docs.worldcubeassociation.org/guides/translating_the_website.html#using-react
+        link: "As a <0>%{pdfLink}</0>"
+      info:
+        download: "Download all of the competitions details."
+      list:
+        empty: "No matching Records found"
+        spots_remaining: "; %{spots} spot(s) remaining"
```

Comment:
Not a big fan of a translation string beginning with a semi-colon.
Is there a more clever way to do this via string interpolation?